frontend/portals/ReplaceMAPComponent/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I think it needs to have different styling depending on the portal position name. Otherwise,
 it looks same everywhere and sometimes it's not so good UX wise.

![screenshot 2019-02-06 at 13 03 56](https://user-images.githubusercontent.com/3393569/52340544-4828fd80-2a10-11e9-98eb-a99c084ca217.png)

Reply:
![screenshot 2019-02-06 at 13 10 25](https://user-images.githubusercontent.com/3393569/52340650-950cd400-2a10-11e9-972f-788796b59888.png)

CG2 D2 has now also configured widget with MAP.

Reply:
I added styling to the category portal to address this issue.

Reply:
Well, it should detect the theme, not the platform ;)

Reply:
made style respond to theme rather than platform.